pom.xml

```diff
@@ -237,6 +237,11 @@
                 <artifactId>jackson-databind</artifactId>
                 <version>${jackson.version}</version>
             </dependency>
+            <dependency>
```

Comment:
jackson-databind don't have jackson-core dependency?

Reply:
Plus it, there won't be a version problem

Reply:
The problem has been submitted many times.

Reply:
Would it be more appropriate to replace a `jackson-databind` version with the correct ` jackson-core` version?